src/connectedk8s/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '1.7.3'
+VERSION = '1.7.4'
```

Comment:
2.0.0. Please refer to extension schema: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md